src/providers/StationsProvider.tsx

```diff
@@ -2,8 +2,24 @@
 import { ReactElement, useContext, createContext, useMemo, useState, useEffect } from "react";
 import { DataService } from "@/services/DataService";
 
+export type Station = {
```

Comment:
It does feel a bit odd to have this type exported here and then use it in the DataService. I would expect all my global types, to be declared in the `@/types/` folder to honest.

Reply:
You're right, I'll move it, thanks